src/ExpressionEvaluator/Core/Source/ExpressionCompiler/ExpressionCompiler.csproj

```diff
@@ -87,10 +87,6 @@
     <DefineConstants>$(DefineConstants);EXPRESSIONCOMPILER</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
```

Comment:
ExpressionCompiler still has a reference to a file in the output directory. Can we fix that to avoid races?